src/worker/dump_db.rs

```diff
@@ -246,6 +253,22 @@ impl Drop for DumpTarball {
     }
 }
 
+struct InvalidateCaches {}
+
+impl InvalidateCaches {
+    pub fn create(client: &Client, target_name: &str) -> Result<(), PerformError> {
+        let cloudfront = CloudFront::from_environment()
+            .context("failed to create CloudFront client from environment")?;
+        cloudfront.invalidate(client, target_name)?;
```

Comment:
using `?` here means that if the cloudfront invalidation fails the fastly invalidation will not run. it might be better to soft-fail here and only log failures as warnings.